api/src/main/java/com/microsoft/gctoolkit/jvm/AbstractJavaVirtualMachine.java

```diff
@@ -179,15 +181,25 @@ private Constructor<? extends Aggregator<?>> constructor(Aggregation aggregation
     @Override
     public void analyze(List<Aggregation> registeredAggregations, JVMEventChannel eventBus, DataSourceChannel dataSourceBus) {
         Phaser finishLine = new Phaser();
-        try {
             Set<EventSource> generatedEvents = diary.generatesEvents();
             for (Aggregation aggregation : registeredAggregations) {
+                if ( debugging)
+                    LOGGER.log(Level.INFO, "Evaluating: " + aggregation.toString());
```

Comment:
Should debug log not be at DEBUG?

Reply:
Tricky.. If the property is set one shouldn't have to set anything else. But, this should be part of a larger conversation about tag vs level based logging.

Reply:
I think this is part of a larger discussion of tags vs levels. My intent in Censum/GCToolKit was to build up a tag based logging framework. This is